sklearn/neural_network/random_activation.py

```diff
@@ -0,0 +1,86 @@
+from abc import ABCMeta, abstractmethod
+
+import numpy as np
+
+from base import ACTIVATIONS
+
+from ..base import TransformerMixin
+from ..externals import six
+from ..utils import check_random_state
+from ..utils.extmath import safe_sparse_dot
+from ..utils.validation import check_array
+
+
+class RandomActivation(six.with_metaclass(ABCMeta, TransformerMixin)):
+    def __init__(self, n_activated_features=10, weight_scale='auto', 
+                 activation='identity', intercept=True, random_state=None):
```

Comment:
Maybe not identity because that would make it the same as RandomProjection more or less?